### ISSUE TITLE
Missing tags

### ISSUE THREAD

**Issue by zklaus** [IS-ENES-Data/QA-DKRZ#19]
There are a number of releases on anaconda for which there is no corresponding tag in this repository. At least for some of them there are commits with a message of tag like
```
commit 4f864e7fe6a0c7ac6b2bf1258a3242d6f471b342
Author: Heinz-Dieter Hollweg <hollweg.dkrz.de>
Date:   Wed Feb 20 13:39:05 2019 +0100

    tag: qa-dkrz-0.6.7-56
```

Perhaps these tags exist in a working copy but where not pushed to github?

Regardless, it would be nice to have tags for all releases available for easy reference in the repository.

**Comment by zklaus** [comment]
I forked the project under https://github.com/zklaus/QA-DKRZ and added the tags to those commits that changed the meta.yaml file.

**Comment by h-dh** [comment]
Sometimes (too often) times were tight and building the conda package on a slow machine and fixing bugs on a different one went in parallel. Thus, there are in deed missing tags. I didn't consider seamless tagging that important.
